interfaces/builtin/kvm.go

```diff
@@ -38,8 +50,58 @@ const kvmConnectedPlugAppArmor = `
 
 var kvmConnectedPlugUDev = []string{`KERNEL=="kvm"`}
 
+type kvmInterface struct {
+	commonInterface
+}
+
+var procCpuinfo = "/proc/cpuinfo"
+
+func getCpuFlags() (flags []string, err error) {
+	buf, err := ioutil.ReadFile(procCpuinfo)
+	if err != nil {
+		// if we can't read cpuinfo, we want to know _why_
+		return nil, fmt.Errorf("unable to read %v: %v", procCpuinfo, err)
+	}
+
+	// want to capture the text after 'flags:' entry
+	pattern := regexp.MustCompile(`(?m)^flags\s+:\s+(.*)$`)
```

Comment:
This should probably be package level `var` so that we don't compile the regex on each call.

Reply:
Done